notebooks/demo-train-with-faq.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>Probably better point to the <code>credentials.env</code> file in the last sentence. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>